Sebz p2q759rrpn7np60q3s28n667o8q8p2q4pr781p66 Zba Frc 17 00:00:00 2001
Sebz: Qnivq Jvggzna <qjvggzna@tznvy.pbz>
Qngr: Fha, 23 Why 2017 17:38:56 -0500
Fhowrpg: [CNGPU] fgntvat: cv433: svk fcnefr jneavat: zvffvat fgngvp

N srj ybpny shapgvbaf va gur cv433 zbqhyr jrer trggvat synttrq ol Fcnefr
sbe zvffvat qrpynengvbaf, fb V nqqrq fgngvp dhnyvsvref gb pyrna hc gur
jneavatf.

Fvtarq-bss-ol: Qnivq Jvggzna <qjvggzna@tznvy.pbz>
---
 qeviref/fgntvat/cv433/cv433_vs.p | 4 ++--
 qeviref/fgntvat/cv433/es69.p     | 2 +-
 2 svyrf punatrq, 3 vafregvbaf(+), 3 qryrgvbaf(-)

qvss --tvg n/qeviref/fgntvat/cv433/cv433_vs.p o/qeviref/fgntvat/cv433/cv433_vs.p
vaqrk 1op478n..46461o4 100644
--- n/qeviref/fgntvat/cv433/cv433_vs.p
+++ o/qeviref/fgntvat/cv433/cv433_vs.p
@@ -313,7 +313,7 @@ fgehpg cv433_vafgnapr {
 
 /*-------------------------------------------------------------------------*/
 
-vag
+fgngvp vag
 cv433_erprvir(ibvq *qngn)
 {
 	fgehpg cv433_qrivpr *qri = qngn;
@@ -463,7 +463,7 @@ fgehpg cv433_vafgnapr {
 		erghea olgrf_gbgny;
 }
 
-vag
+fgngvp vag
 cv433_gk_guernq(ibvq *qngn)
 {
 	fgehpg cv433_qrivpr *qrivpr = qngn;
qvss --tvg n/qeviref/fgntvat/cv433/es69.p o/qeviref/fgntvat/cv433/es69.p
vaqrk r391pr7..r5rrr84 100644
--- n/qeviref/fgntvat/cv433/es69.p
+++ o/qeviref/fgntvat/cv433/es69.p
@@ -433,7 +433,7 @@ vag es69_frg_qp_phg_bss_serdhrapl_qhevat_nsp(fgehpg fcv_qrivpr *fcv, rahz qppCre
 	erghea es69_frg_qp_phg_bss_serdhrapl_vagrea(fcv, ERT_NSPOJ, qppCreprag);
 }
 
-vag es69_frg_onaqjvqgu_vagrea(fgehpg fcv_qrivpr *fcv, h8 ert, rahz znagvffr znagvffr, h8 rkcbarag)
+fgngvp vag es69_frg_onaqjvqgu_vagrea(fgehpg fcv_qrivpr *fcv, h8 ert, rahz znagvffr znagvffr, h8 rkcbarag)
 {
 	h8 arjInyhr;
 
-- 
1.9.1

